btb-phylo

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Suggest renaming as `btb-phylo.sh` to indicate that this is a bash script

Reply:
Cool, done